backend/src/jobs.rs

```diff
@@ -1416,7 +1445,12 @@ pub async fn pull(db: &DB) -> Result<Option<QueuedJob>, crate::Error> {
             WHERE id IN (
                 SELECT id
                 FROM queue
-                WHERE running = false AND scheduled_for <= now()
+                WHERE running = false
+                  AND scheduled_for <= now()
+                  AND (   suspend = ( SELECT count(*)
+                                        FROM resume_job r
+                                       WHERE queue.id = r.job )
```

Comment:
I thought we agreed to use a counter that we would increase when we reach the suspend, decrease when we get a resume event and do suspend = 0 here ?

Reply:
Yes that would be preferable, hadn't 100% quite worked out how to do that safely when I wrote the above.